test/unit/view.dynamic-regions.spec.js

```diff
@@ -14,22 +14,12 @@ describe('itemView - dynamic regions', function() {
 
   describe('when adding a region to a layoutView, after it has been rendered', function() {
     beforeEach(function() {
-      this.MyView = Marionette.View.extend({
-        onAddRegion: function() {},
-        onBeforeAddRegion: function() {}
-      });
+      this.MyView = Marionette.View.extend();
```

Comment:
all of the test changes in this file relate to removing `add:region` / `remove:region`